lib/features/base/mixin/compose_floating_button_mixin.dart

```diff
@@ -0,0 +1,56 @@
+
+import 'package:core/presentation/extensions/color_extension.dart';
+import 'package:core/presentation/resources/image_paths.dart';
+import 'package:core/presentation/utils/style_utils.dart';
+import 'package:core/presentation/views/floating_button/scrolling_floating_button_animated.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:tmail_ui_user/main/localizations/app_localizations.dart';
+import 'package:tmail_ui_user/main/routes/route_navigation.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+mixin ComposeFloatingButtonMixin {
```

Comment:
Why do you have to use mixin here, why just use StatelessWidget ?

Reply:
Oh, I forgot to fix it. We can use it as a stateless